tests/ui/type-alias-impl-trait/issue-109054.rs

```diff
@@ -10,8 +10,10 @@ type FnType = impl Fn(&u32) -> ReturnType;
 impl std::ops::Deref for CallMe {
     type Target = FnType;
 
+    #[define_opaques(FnType)]
     fn deref(&self) -> &Self::Target {
         //~^ ERROR: item does not constrain `ReturnType
```

Comment:
should APTIT collection auto collect TAITs? I would expect no. Please add a FIXME/note somewhere

Reply:
it doesn't. `define_opaques(FnType)` does. There is already a

> // FIXME(type_alias_impl_trait): can we require mentioning nested type aliases explicitly?